New_PlatformGame/Assets/UIScript.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Net.Mime;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UIScript : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public Text scoreText;
+    private int _score;
+    public GameObject player;
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        SetScore();
+    }
+
+    void SetScore()
+    {
+        _score = player.GetComponent<GameManager>().score;
+        scoreText.text = _score + "M";
```

Comment:
```suggestion
        scoreText.text = _score + "M";
```

당장 개선이 필요한 부분은 아니지만, C#의 string은 + 연산자를 이용할 때 새로운 string 객체를 만들기 때문에 + 연산이 끝나면 가비지 컬렉터를 호출한다. 이를 개선하기 위해 C#에서는 StringBuilder라는 클래스를 지원하는데, 이에 대해 공부해보면 C#에 대해 공부하기 좋을 것.
https://docs.microsoft.com/ko-kr/dotnet/api/system.text.stringbuilder?view=net-5.0